### DISCUSSION TITLE
Prism WPF - Mend Security Scan Vulnerabilities Issues Fix Request

### DISCUSSION THREAD

**Discussion by karthikeyanthangaraju** [PrismLibrary/Prism#2982] [WPF]
Hello,

I am writing to request your assistance regarding the Prism WPF Nuget package (Version: 8.1.97) that we are currently using. We recently conducted Mend Security scans which identified some vulnerabilities in the Prism WPF library. As a result, we would like to request your help in addressing the vulnerabilities listed below and obtaining the latest version of Prism WPF. 

Thank you for your attention to this matter.

As part of the scan, we have identified below four vulnerabilities.

![image](https://github.com/PrismLibrary/Prism/assets/126872854/13869dd1-b912-44b7-b3dc-9bb0068c886d)
![image](https://github.com/PrismLibrary/Prism/assets/126872854/0e67b8e3-129c-4108-9df4-38f6e782cc45)
![image](https://github.com/PrismLibrary/Prism/assets/126872854/feee62ff-8c51-453e-aff1-e20439f29db1)
![image](https://github.com/PrismLibrary/Prism/assets/126872854/ffcc2f28-da6a-4150-8818-a0d2a57a6ef7)

**Comment by brianlagunas** [comment]
I can't tell if this is a spam post or not, but I will give you the benefit of the doubt.  The images you have provided are not vulnerabilities. They are dependency graphs.

**Reply by dansiegel** [reply]
I will add here that the dependency graphs that you have provided are only relevant if you are targeting net461, updating your project to net47 or later would actually remove the extra dependencies in the graph. Also worth noting that you may be picking things up as a vulnerability as net461 fell out of support last year.
https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework

**Reply by karthikeyanthangaraju** [reply]
Thanks, Dan and Brain for your quick response.
